### DISCUSSION TITLE
Cassandra Backend

### DISCUSSION THREAD

**Discussion by decibelcooper** [cschleiden/go-workflows#249] [Q&A] [answered]
I've just discovered this project, but I'm curious enough about the feasibility of having a Cassandra `Backend` to try to make a PoC and perhaps a contribution.  It would be neat to try to achieve not only linear scalability, but also multi-datacenter, multi-active operation.

I'd appreciate some feedback on perceived challenges here, and whether or not there are any anticipated changes to the `Backend` interface that may have a significant impact (also whether or not suggested changes to accommodate such a backend would be welcome).

**Comment by cschleiden** [comment, marked as answer]
I think that should be possible. I'm not familiar with Cassandra but the `backend.Backend` interface is loosely modeled after Azure's Durable Task one and I know there are various implementations that use the different Azure database offers. 

If you want to attempt it and run into any roadblocks with the interface, lmk.

**Reply by decibelcooper** [reply]
Thanks for the feedback!
